packages/studio-be/src/core/dialog/flow/flow-service.ts

```diff
@@ -174,6 +174,11 @@ export class ScopedFlowService {
     const expectedSaves = this.expectedSavesCache.get(flowPath)
 
     if (!expectedSaves) {
+      // fix an issue when creating a bot where the .flow.json is written but not the .ui.json because of the locking mechanism
+      if (!(await this.ghost.fileExists(FLOW_DIR, this.toUiPath(flowPath)))) {
+        return
+      }
+
```

Comment:
This was added in the ghost of Botpress, but we forgot to copy it in the studio's ghost.